android_test/Android.mk

```diff
@@ -5,7 +5,7 @@ LOCAL_CPP_EXTENSION := .cc .cpp .cxx
 LOCAL_SRC_FILES:=test.cpp
 LOCAL_MODULE:=spirvtools_test
 LOCAL_LDLIBS:=-landroid
-LOCAL_CXXFLAGS:=-std=c++11 -fno-exceptions -fno-rtti -Werror
+LOCAL_CXXFLAGS:=-std=c++17 -fno-exceptions -fno-rtti -Werror
```

Comment:
Heh. That might explain some things.... :-)
Thank you!

Reply:
For some reasons the macos debug CI failed with this change (but passes at HEAD).
Does this rings a bell? (unordered_map is not defined).
Restarted it, a bit weird.